atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/ProfilingCassandraClient.java

```diff
@@ -69,12 +70,18 @@ public ProfilingCassandraClient(CassandraClient client) {
         return KvsProfilingLogger.maybeLog(
                 (KvsProfilingLogger.CallableCheckedException<Map<ByteBuffer, List<ColumnOrSuperColumn>>, TException>)
                         () -> client.multiget_slice(kvsMethodName, tableRef, keys, predicate, consistency_level),
-                (logger, timer) -> logger.log("client.multiget_slice({}, {}, {}, {}) on kvs.{}",
+                (logger, timer) -> logger.log("CassandraClient.multiget_slice({}, {}, {}, {}) on kvs.{} took {} ms",
                         LoggingArgs.tableRef(tableRef),
-                        SafeArg.of("number of keys", numberOfKeys),
-                        SafeArg.of("number of columns", numberOfColumns),
+                        LoggingArgs.rowCount(numberOfKeys),
+                        LoggingArgs.columnCount(numberOfColumns),
                         SafeArg.of("consistency", consistency_level.toString()),
-                        SafeArg.of("kvsMethodName", kvsMethodName)));
+                        SafeArg.of("kvsMethodName", kvsMethodName),
+                        LoggingArgs.durationMillis(timer)),
+                (logger, rowsToColumns) -> logger.log("and returned {} cells in {} rows with {} bytes",
```

Comment:
Why do we split up these log lines?

Reply:
I don't know, but it is very annoying to read these logs.  If we can combine them during this PR it would be very helpful.

Reply:
1. Note that this does not create two log lines. Our logger is a bit special here, and just concatenates these two log lines, printing just once, when the call to Cassandra returns with either a result or an exception.

2. We need to have two methods because the second one is called when the call to Cassandra returns and we have access to the result of the call. The call might fail, and in this case we concatenate an "and failed with exception {}" at the end of the log line.